docs/l10n.rst

```diff
@@ -910,144 +910,3 @@ from the lang files then you can use the `add_active_locales` name in all of the
-need translation.
-
-.. note::
-    For source Fluent files currently...
-
-    * ...Mozorg uses ``./l10n/``
-    * ...Pocket uses ``./l10n-pocket/``
-
-Add the following files:
-
-.. code-block:: bash
-
-    ./l10n-newintegration/
-    ./l10n-newintegration/en/  # This is where source Fluent templates go for 'newintegration'
-    ./l10n-newintegration/en/configs/pontoon.toml  # If using community/Pontoon translations at all
-    ./l10n-newintegration/en/configs/vendor.toml  # If using a paid-for translation service such as Smartling
-    ./l10n-newintegration/en/configs/special-templates.toml   # Only needed to exclude certain files from all community AND vendor translation. e.g. we use staff translation only.
-
-    ./l10n-newintegration/l10n-pontoon.toml  # If using community/Pontoon translations at all
-    ./l10n-newintegration/l10n-vendor.toml  # If using a paid-for translation service such as Smartling
-
-    ./data/l10n-newintegration-team/  # leave this empty - it will be populated via a git sync using data FROM the l10n team
-
-For the exact content of each `.toml` or `.json` file, see the examples in
-``./l10n/`` and ``./l10n-pocket/`` for inspiration - they're not too hard to work out.
-The ``.toml`` files outside of ``/en/`` basically point to the ones in ``/en/configs/``
-and are a 'gateway' through which we spec which config files are relevant to which
-translation stragegy (community or vendor - or neither if it's staff-only translation).
-
-2. **REPO SETUP (Bedrock and/or L10N team admin)**
-
-You will need to set up one or two new repos, to hold the translation files as
-part of the pipeline.
-
-i. **A repo in where the files are sent to** in ``https://github.com/mozilla-l10n/``
-for the L10N team's automation to pick up.
-
-For example, Mozorg uses ``github.com/mozilla-l10n/www-l10n/`` and Pocket uses
-``github.com/mozilla-l10n/www-pocket-l10n/``.
-Your new ``github.com/mozilla-l10n/www-newintegration-l10n/`` repo will be needed
-regardless of who does the actual translation work.
-
-ii. **An optional repo where files are post-processed following translation**.
-
-If relevant, this will live in ``github.com/mozmeao/`` - for example ``github.com/mozmeao/www-newintegration-l10n/``
-
-.. important::
-    **If you are not using Pontoon/community translations, you do NOT need to create
-    this repo.**
-    Why? If the translations are done by the community (via Pontoon), there is a
-    possibility that not enough of the strings will be translated in order to render
-    the content in the relevant locale. We run a :abbr:`CI (Continuous Integration)`
-    task to determine whether a locale has enough translated strings to be considered
-    'active'. At the moment, only Mozorg uses this pattern. The Pocket-mode translations
-    do not have their activation measured because their translations come entirely from
-    a vendor and we expect the Pocket strings to be 100% translated.
```

Comment:
I added this back and re-wrote it more as informational docs about how the l10n pipeline works instead of instructions on adding a new pipeline. Thanks for pointing out the usefulness of this section.